tests/main/snap-seccomp-syscalls/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: Ensure that we accounted for all syscalls known to libseccomp
+
+# one system is enough
+systems: [ubuntu-18.04-64]
```

Comment:
shouldn't this be always the newest ubuntu system we test on?

Reply:
We're building the latest master. By the looks of libseccomp's git log, they track the kernel quite closely and the syscalls list they maintain does not depend on the host system.

Reply:
thanks for the explanation